recipes/sphinx-markdown-tables/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "sphinx-markdown-tables" %}
+{% set version = "0.0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 08779e77832b6562b6b86e4270ae50b0b10010fdd35cf9aadcb5b8246571b933
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - markdown ==3.0.1
+    - pip
+    - python
+  run:
+    - markdown ==3.0.1
+    - python >=3.5
+    - recommonmark
+
+test:
+  imports:
+    - sphinx_markdown_tables
+  requires:
+    - markdown ==3.0.1
```

Comment:
You don't need the packages that are in the `run` section listed here

Reply:
Thanks. I have slimmed down both the host and test requirements.